tests/test_orquestra_device.py

```diff
@@ -0,0 +1,293 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+import subprocess
+import os
+import uuid
+import time
+import numpy as np
+
+import pennylane as qml
+import pennylane.tape
+import pennylane_orquestra
+from pennylane_orquestra import OrquestraDevice, QeQiskitDevice, QeIBMQDevice
+from conftest import (
+    test_batch_res0,
+    test_batch_res1,
+    test_batch_res2,
+    resources_default,
+    MockPopen,
+)
+
+qiskit_analytic_specs = '{"module_name": "qeqiskit.simulator", "function_name": "QiskitSimulator", "device_name": "statevector_simulator"}'
+qiskit_sampler_specs = '{"module_name": "qeqiskit.simulator", "function_name": "QiskitSimulator", "device_name": "statevector_simulator", "n_samples": 1000}'
+forest_noisy_specs = '{"module_name": "qeforest.simulator", "function_name": "ForestSimulator", "device_name": "3q-noisy-qvm", "n_samples": 10000}'
+ibmq_specs = '{"module_name": "qeqiskit.backend", "function_name": "QiskitBackend", "device_name": "ibmq_qasm_simulator", "n_samples": 1000, "api_token": "Some token"}'
```

Comment:
Perhaps the fact that 10000 shots are needed here is a sign that we should increase the default number of shots on the devices? Although I _think_ Qiskit only supports max 8192 shots

Reply:
this is a very minor suggestion though, I'm happy for it to be left as is

Reply:
That sounds like a good idea, could increase to 10000 shots for simulators and 8192 for IBMQ.